docs/terminal/sydney.md

```diff
@@ -14,20 +14,19 @@
 | Sydney Departures South†   |SDS| Sydney Departures | 129.700          | SY-S_DEP         |
 | Sydney Director West† |SFW| Sydney Director   | 126.100          | SY-D_APP                               |
 | Sydney Director East† |SFE| Sydney Director   | 125.300          | SY-DE_APP                               |
-| Sydney Radar† |SRI| Sydney Centre  | 124.550          | SY-C_DEP                               |
+| Sydney Radar†* |SRI| Sydney Centre  | 124.550          | SY-C_DEP                               |
 | Sydney Flow†        |SFL|                |          | SY-FLW_CTR                               |
 
-† *Non-standard positions* may only be used in accordance with [VATPAC Ratings and Controller Positions Policy](https://vatpac.org/publications/policies)
+† *Non-standard positions* may only be used in accordance with [VATPAC Ratings and Controller Positions Policy](https://vatpac.org/publications/policies).  
+* [Additional requirements](#airspace-structural-arrangements) must be met prior to opening SRI as a stand-alone position.
```

Comment:
You could use `‡` (the double dagger) instead of `*` here - as the `*` gets interpreted as a bullet point in Markdown.

Reply:
@oliverfrye it's displaying as desired due to the space between the * and the sentence, the double dagger is displaying full size rather than half size (like the single dagger).  I'm happy to replace it, do you have a link to a small double dagger character I can use instead?

Reply:
It looks like this in the GitHub preview - it might be different when it actually gets rendered by `mkdocs` so if you've checked it in the rendered view already then please disregard.

<img width="211" alt="image" src="https://user-images.githubusercontent.com/11541660/222608443-522b1b40-a3e1-4a83-9907-5dd560604b44.png">

If I add the double dagger it looks like this for me - is this different to what you're seeing?

<img width="199" alt="image" src="https://user-images.githubusercontent.com/11541660/222608666-4b8afa21-1bdb-450c-8703-bfd8339ac9b4.png">

Reply:
mkdocs is rendering this:
![image](https://user-images.githubusercontent.com/41880668/222609156-e6689b3b-736c-47ec-a027-21942dcdf5a8.png)

or this:
![image](https://user-images.githubusercontent.com/41880668/222609238-7ccfa884-b4d4-4742-9fce-e614026415a0.png)

Reply:
Oh, wow, I see - please disregard with my apologies... looks like there's a font issue with the double dagger in the `mkdocs` rendering, it must be falling back to a different font.